src/main/java/seedu/address/model/interview/UniqueInterviewList.java

```diff
@@ -70,6 +71,34 @@ public void setInterview(Interview target, Interview editedInterview) throws Dup
         internalList.set(index, editedInterview);
     }
 
+    /**
+     * Sorts the interviews by date and time.
+     */
+    public void sortInterviewsByDate() {
+        Comparator<Interview> dateSorter = (interview1, interview2) -> {
```

Comment:
Good choice of creating the anonymous `Comparator` in a separate variable instead of defining it directly in the `internalList.sort()` call. Improves readability